### PR TITLE
Test prisma

### PR DESCRIPTION
**Pull request by xTye**
Please look at the changed files and leave any comments or suggestions I can make to the existing branch. I will continue to make changes until no more comments have been issued. I won't merge until yall approve this merge

### CODE REVIEW

File: src/lib/prisma.ts [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+  var prisma: PrismaClient | undefined
+}
+
+export const prisma =
+  global.prisma ||
+  new PrismaClient({
+    log: ['query'],
+  })
+
+if (process.env.NODE_ENV !== 'production') global.prisma = prisma
```
Comment (xTye) [inline src/lib/prisma.ts:15]: add end of line

**Comment by Gian-Alvarez** [comment]
All code looks good.

**Review by Gian-Alvarez — APPROVED** [review]
All code looks good.

**Review by xTye — COMMENTED** [review]
sdf

**Pull request merged**